test/TestUtilities/Test.Utility/Signing/TestCertificateGenerator.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Test.Utility.Signing
+{
+    public class TestCertificateGenerator
+    {
+        public static readonly Oid IdKPCodeSigning = new Oid("1.3.6.1.5.5.7.3.3");
+        public static readonly Oid IdKPClientAuth = new Oid("1.3.6.1.5.5.7.3.2");
+        public static readonly Oid IdKPEmailProtection = new Oid("1.3.6.1.5.5.7.3.4");
+        public static readonly Oid AnyExtendedKeyUsage = new Oid("2.5.29.37");
```

Comment:
This is wrong.  is 2.5.29.37.

Search [RFC 5280](https://tools.ietf.org/html/rfc5280) for `anyExtendedKeyUsage`.

Also see http://oid-info.com/cgi-bin/display?oid=2.5.29.37.0&action=display